editor/src/clj/editor/camera_editor.clj

```diff
@@ -129,14 +133,14 @@
 
 (def outline-shader (shader/make-shader ::outline-shader outline-vertex-shader outline-fragment-shader))
 
-(defn- gen-outline-vertex [x y z w cr cg cb]
-  [(/ x w) (/ y w) (/ z w) cr cg cb 1])
+(defmacro ^:private gen-outline-vertex [x y z w cr cg cb]
+  `[(/ ~x ~w) (/ ~y ~w) (/ ~z ~w) ~cr ~cg ~cb 1.0])
 
 (defn- conj-camera-mesh-vertices! [vbuf ^Matrix4d inv-view cr cg cb]
   (mapv (fn [^Vector4d p]
           (let [tp (math/transform-vector-v4 inv-view p)
                 camera-mesh-vx (gen-outline-vertex (.x tp) (.y tp) (.z tp) (.w tp) cr cg cb)]
-            (-> vbuf (conj! camera-mesh-vx))))
+            (conj! vbuf camera-mesh-vx)))
```

Comment:
We typically only resort to the threading macro (`->`) when there are several steps.